### PR TITLE
Some steps to fix Travis CI

### PR DESCRIPTION
**Pull request by AlekSi**
Notice that Travis CI fails for some time: https://github.com/grpc-ecosystem/grpc-gateway/pulls
That PR allows it to move farther.

### CODE REVIEW

**Comment by tmc** [comment]
kick clabot

**Comment by tmc** [comment]
@AlekSi have you signed the CLA?

**Comment by tmc** [comment]
as per https://github.com/grpc-ecosystem/grpc-gateway/pull/434 this contributor has signed the CLA

File: .travis.yml [hunk 1]
```diff
@@ -28,7 +29,7 @@ script:
 - if (go version | grep -q 1.8) && [ -z "${GATEWAY_PLUGIN_FLAGS}" ]; then test -z "$(git status --porcelain)" || (git status; git diff; exit 1); fi
 - env GLOG_logtostderr=1 go test -race -v github.com/grpc-ecosystem/grpc-gateway/...
 - make lint
-- sh -c 'cd examples/browser && gulp'
+- sh -c 'cd examples/browser && node ./node_modules/gulp/bin/gulp'
```
Comment (tmc) [inline .travis.yml]: can we do $(npm bin)/gulp?

**Comment by achew22** [comment]
Looks like you've [previously contributed](https://github.com/grpc-ecosystem/grpc-gateway/pull/434) and everything was good with that, plus the CLA bot is kind of on the fritz. I'm going to merge this.

**Comment by AlekSi** [comment]
Just to confirm: yes, CLA is signed.

**Pull request merged**